lib/shopify_transporter/pipeline/magento/order/addresses_attribute.rb

```diff
@@ -21,7 +22,22 @@ def get_nested_address(input, prefix)
             items = input['items']
             result = items && items['result']
             address = result && result["#{prefix}address"]
-            address || []
+            address ||= []
+            purge_address(input, address, prefix)
+          end
+
+          def purge_address(input, address, prefix)
```

Comment:
I prefer a different word here than `purge` - what do you think of `clean_address` or `filter_address`? Purge makes me think that we're deleting the entire address, but we're purging nested parts of it.